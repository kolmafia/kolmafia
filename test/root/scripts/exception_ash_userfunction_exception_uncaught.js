const { print } = require("kolmafia");
const helper = require("exception_ash_userfunction_helper.ash");

print(helper.returnsEleven());
helper.throwScriptException();
print("This code should not be reached.");