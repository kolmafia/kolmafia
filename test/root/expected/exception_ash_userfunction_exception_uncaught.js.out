11
Script exception: Can't get session logs for a negative number of days (exception_ash_userfunction_helper.ash, line 10)